wcvendors/wcvendors
# Fixed: Order Notification for the vendor does not include the coupon …

…discount

### All Submissions:






### Changes proposed in this Pull Request:



Closes #848 
- Added coupon line to vendor notification

### How to test the changes in this Pull Request:

1. Create a coupon
2. Place an order using that coupon
3. Check the notifications received by both the admin and the vendor

### Other information:
